CRM/Utils/Check/Component/Env.php

```diff
@@ -453,16 +453,16 @@ public function checkDirsWritable() {
     if (!empty($notWritable)) {
       $messages[] = new CRM_Utils_Check_Message(
         __FUNCTION__,
-        ts('The %1 is not writable.  Please check your file permissions.', array(
+        ts('The %1 is not writable. Writing to Settings file has been disabled by admin. Please check your file permissions.', array(
           1 => implode(', ', $notWritable),
           'count' => count($notWritable),
-          'plural' => 'The following directories are not writable: %1.  Please check your file permissions.',
+          'plural' => 'The following directories are not writable: %1. Writing to Settings file has been disabled by admin. Please check your file permissions.',
         )),
         ts('Directory not writable', array(
           'count' => count($notWritable),
           'plural' => 'Directories not writable',
         )),
-        \Psr\Log\LogLevel::ERROR,
+        \Psr\Log\LogLevel::NOTICE,
```

Comment:
This should remain an `error`: it's a real problem if CiviCRM can't upload import files, images, or custom files.  In nearly all cases, a site that can't write to these folders will appear "broken" to someone trying to do regular tasks like running imports or uploading a file to a custom field.